docker-compose.yml

```diff
@@ -1,81 +1,59 @@
-version: '2.1'
-
-networks:
-  get5:
-    external: true
+version: "3.7"
 
 services:
-  caddy:
-    image: caddy:2-alpine
-    container_name: caddy-reverse-proxy
-    restart: unless-stopped
+  redis:
+    image: redis:6
+    container_name: redis
+    restart: always
     ports:
-      - "80:80"
-      - "443:443"
-    volumes: 
-      - ./Caddyfile:/etc/caddy/Caddyfile
-    networks:
-      - get5
-    environment:
-      - CADDY_URL=
-      - CADDY_API_ENDPOINT=api
-      - CADDY_REVERSE_PROXY_PORT=3301
-  
+      - 6379:6379
+
   get5db:
-    image: mariadb
+    image: mysql
     container_name: get5db
     restart: always
     environment:
-      - MYSQL_ROOT_PASSWORD=
+      - MYSQL_ROOT_PASSWORD=testing
       - MYSQL_DATABASE=get5
       - MYSQL_USER=get5
-      - MYSQL_PASSWORD=
+      - MYSQL_PASSWORD=testing
     ports:
-      - "3306:3306"
-    networks:
-      - get5
+      - 3306:3306
 
   g5api:
-    image: ghcr.io/phlexplexico/g5api:latest
```

Comment:
These should be built from the container registry instead of local, so that users will get a stable build from what exists on master. If a user wishes to change to build local, they can do so themselves I feel. This docker-compose is meant as a "quickstart" for those who aren't really experienced with development and just need a solution (i.e. tournament managers).

Reply:
Yea that sounds reasonable. I had used local images as the apiurl is dependant on the G5V image build (as it is a build arg) and by using local images caddy could be removed since it only points the 3301 port to /api which is unnecessary if the apiurl is set to use the 3301 port instead of /api. But I agree that if the docker-compose is supposed to be quickstart then caddy and container registry images are needed